<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardiacCT</class>
 <widget class="qMRMLWidget" name="CardiacCT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>941</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="loadImagesButton">
       <property name="accessibleName">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Load - Images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveSegmentationsButton">
       <property name="styleSheet">
        <string notr="true">background: #008c05; color:black;</string>
       </property>
       <property name="text">
        <string>Save - Segmentations</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Anatomical</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="ctkCollapsibleButton" name="CollapsibleButtonCTA">
         <property name="styleSheet">
          <string notr="true">ctkCollapsibleButton { background: #ed0000; color:black;}</string>
         </property>
         <property name="text">
          <string>CTA</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="qMRMLSegmentEditorWidget" name="CTAEditorWidgetAnatomical">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="defaultTerminologyEntrySettingsKey">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="ctkCollapsibleButton" name="CollapsibleButtonCACS">
         <property name="styleSheet">
          <string notr="true">ctkCollapsibleButton { background: #feda00; color:black;}</string>
         </property>
         <property name="text">
          <string>CACS</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="contentsLineWidth">
          <number>-2</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="qMRMLSegmentEditorWidget" name="CACSEditorWidgetAnatomical">
            <property name="defaultTerminologyEntrySettingsKey">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Lesions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="ctkCollapsibleButton" name="CollapsibleButtonCACS2">
         <property name="styleSheet">
          <string notr="true">ctkCollapsibleButton { background: #ed0000; color:black;}</string>
         </property>
         <property name="text">
          <string>CTA</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="qMRMLSegmentEditorWidget" name="CTAEditorWidgetLesions">
            <property name="defaultTerminologyEntrySettingsKey">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="ctkCollapsibleButton" name="CollapsibleButtonCTA2">
         <property name="styleSheet">
          <string notr="true">ctkCollapsibleButton { background: #feda00; color:black;}</string>
         </property>
         <property name="text">
          <string>CACS</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="qMRMLSegmentEditorWidget" name="CACSEditorWidgetLesions">
            <property name="defaultTerminologyEntrySettingsKey">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLSegmentEditorWidget</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLSegmentEditorWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
